ndcube/tests/test_utils_cube.py

```diff
@@ -88,7 +88,7 @@ def test_select_order():
     ([('name', 0)], [False, False], (1, 2)),
     ([(0, 0, 0)], [False, False], (1, 2)),
     ([('name', '0', 0)], [False, False], (1, 2)),
-    ([('name', 0, [0, 1])], [False, False], (1, 2))
+    ([('name', 0, [0, 1], 1)], [False, False], (1, 2))
```

Comment:
This set of inputs should be test this [line](https://github.com/sunpy/ndcube/blob/master/ndcube/utils/cube.py#L96) in ```_format_input_extra_coords_to_extra_coords_wcs_axis```.  Therefore the length of the ```coord[2]``` must not equal ```data_shape[1]```.  This is not the case as ```len([0, 1])``` does equal ```2```.  Therefore change the input to this:
```python
([('name', 0, [0, 1, 2])], [False, False], (1, 2))
```